CONTRIBUTING.md

```diff
@@ -242,6 +242,10 @@ who you are:
 $ git config --global user.name "J. Random User"
 $ git config --global user.email "j.random.user@example.com"
 ```
+Please make sure this local email is also added to your
+[GitHub email list](https://github.com/settings/emails) so that your commits
+will be properly associated with your account and you will be timely promoted
+to contributors once your first commit is landed.
```

Comment:
Maybe `to contributors` -> `to Contributor` (it's a title I think).

Reply:
And done,